src/data-publication/reports/tables/Aggregate2.jsx

```diff
@@ -46,95 +84,163 @@ const Aggregate2 = React.forwardRef((props, ref) => {
     return 0
   })
 
+  return fixUnaggregatedValue(sorted)
+}
+
+export const buildCSVRowsAggregate2 = (report) => {
+  let theCSVRows = ''
+
+  const _data = sortAndFix(report)
+  
+  _data.forEach(tract => {
+    const _columns = tract.values.sort(function(x,y){
+      var xp = x.dispositionName.substring(x.dispositionName.length-3);
+      var yp = y.dispositionName.substring(y.dispositionName.length-3);
+      return xp == yp ? 0 : xp < yp ? -1 : 1;
+    }).map(val => [val.count, val.value]).flat().join(',')
+
+    theCSVRows += tract.tract + ',' + _columns + '\n'
+  })
+
+  return theCSVRows
+}
+
+const formatDisplayLabel = (items) => {
```

Comment:
Second case where we are using the `formatDisplayLabel` function.

Reply:
Moved into util file.